src/providers/python.rs

```diff
@@ -305,6 +313,14 @@ impl PythonProvider {
                 .ok_or_else(|| anyhow::anyhow!("failed to load pyproject.toml"))?),
         ))
     }
+
+    fn uses_numpy(app: &App) -> Result<bool> {
+        Ok(app.includes_file("requirements.txt")
+            && app
+                .read_file("requirements.txt")?
+                .to_lowercase()
+                .contains("numpy"))
```

Comment:
This should probably support things like poetry.lock

Reply:
ya good point?